pom.xml

```diff
@@ -87,6 +91,11 @@
 			<artifactId>jaydio</artifactId>
 			<version>0.1</version>
 		</dependency>
+		<dependency>
+		  <groupId>com.google.googlejavaformat</groupId>
+		  <artifactId>google-java-format</artifactId>
+		  <version>1.9</version>
+		</dependency>
```

Comment:
What is this for? Is this necessary to be in dependencies?

Reply:
I definitely forgot to remove this dependency for some reasons
I've made a new commit and remove these lines